python-avd/pyavd/_eos_designs/structured_config/overlay/ip_security.py

```
@@ -36,9 +36,6 @@ def ip_security(self: AvdStructuredConfigOverlayProtocol) -> None:
         if not self.inputs.wan_ipsec_profiles:
             msg = "wan_ipsec_profiles"
             raise AristaAvdMissingVariableError(msg)
-        if not self.inputs.wan_ipsec_profiles.control_plane:
```

Why we are removing this, a user can set the data_plan only in wan_ipsec_profiles so we may need to raise this error

---

because `control_plane` is a required key and it already throws error in validation if its not defined